src/Mod/PartDesign/App/FeaturePolarPattern.cpp

```diff
@@ -48,12 +48,17 @@ namespace PartDesign {
 
 PROPERTY_SOURCE(PartDesign::PolarPattern, PartDesign::Transformed)
 
+const App::PropertyIntegerConstraint::Constraints intOccurrences = { 1, INT_MAX, 1 };
+const App::PropertyAngle::Constraints floatAngle = { FLT_MIN, 360, 1.0 };

```

Comment:
Defining the minimum of FLT_MIN, i.e. ~1.1e-38 doesn't make much sense because practically you cannot set such a small value in the GUI and in the execute() method there is another check for Precision::Confusion() which is 1e-7.
However, using Precision::Confusion() for angles isn't the best idea either since OCCT provides a special tolerance for angles: Precision::Angular() which is 1e-12.

This value, however, is given in radians and not degrees. So, the minimum can be defined as
`Base::toDegrees<double>(Precision::Angular())` or hard-coded as 6.0e-11

The addi

Reply:
Many thanks! I updated the PR accordingly. i thin it is better not to hard-code in case OCCT changes the tolerance in future.

While making the PR I realized that many PD features miss some constraints. I will try to improve the situation bit by bit the next days.